network/network_windows.go

```diff
@@ -78,6 +79,10 @@ func (nm *networkManager) newNetworkImpl(nwInfo *NetworkInfo, extIf *externalInt
 		extIf:     extIf,
 	}
 
+	// Sleep for 10 seconds as a workaround for windows
+	// This is done only when the network is created.
+	time.Sleep(10 * 1000)
```

Comment:
I think you may need something like this to sleep for 10 seconds.
"time.Sleep(time.Duration(10) * time.Second)"